File: 1. Begginer/12- Pida al usuario un número entero, diga si es par o impar,y repita todo lo anterior hasta que el número que se introduzca sea 0. (Nota para saber si un número es par, se mira si el resto de s/main.py
num = 1
while num > 0:
	num = int(input("Ingrese un número entero: "))
	if num == 0 :
		print("El programa ha finalizado")
		break
	else :
		i = 0
		if num%2 == 0 :
			print("Es par")
		else :
			print("Es impar")

File: 1. Begginer/12- Pida al usuario un número entero, diga si es par o impar,y repita todo lo anterior hasta que el número que se introduzca sea 0. (Nota para saber si un número es par, se mira si el resto de s/main.go
package main

import "fmt"

func main() {
	var num int

	for i:=0; i < 2; i++ {
		fmt.Println("Ingrese un número entero: \n")
		fmt.Scanf("%d", &num)
		if num == 0 {
			fmt.Println("El programa ha finalizado")
			break
		} else {
			i = 0
			if num%2 == 0 {
				fmt.Println("El número es par\n")
			} else {
				fmt.Println("El número es impar\n")
			}
		}
	}
}